Project.toml

```diff
@@ -38,10 +39,11 @@ YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 
 [weakdeps]
 Geant4 = "559df036-b7a0-42fd-85df-7d5dd9d70f44"
-LegendHDF5IO="c9265ca6-b027-5446-b1a4-febfa8dd10b0"
+LegendHDF5IO = "c9265ca6-b027-5446-b1a4-febfa8dd10b0"
+LightXML = "9c8b4983-aa76-5018-a973-4c85ecc9e179"
 
 [extensions]
-SolidStateDetectorsGeant4Ext = "Geant4"
+SolidStateDetectorsGeant4Ext = ["Geant4", "LightXML"]
```

Comment:
Does it make sense to make `LightXML` an actual dependency to avoid having to load both `Geant4` and `LightXML` to load the extension?

Reply:
Yes, let's make it a hard dependency, then the GDML export can also be a standard feature. LightXML won't add noticeably to SSD's load time.